packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -178,7 +177,7 @@ describe("BnsConnection", () => {
     expect(addrAcct.name).toEqual("admin");
     expect(addrAcct.balance.length).toEqual(1);
     expect(addrAcct.balance[0].tokenTicker).toEqual(cash);
-    expect(addrAcct.balance[0].whole).toBeGreaterThan(1000000);
+    expect(Number.parseInt(addrAcct.balance[0].quantity, 10)).toBeGreaterThan(1000000_000000000);
```

Comment:
In JavaScript `_` is an optional separator that has no meaning other then helping readability. I think this can be context specific. In Lisk e.g. you might want to express 3, 5, 14 LSK as 3_00000000, 5_00000000, 14_00000000 but 60 seconds can be expressed as 60_000 ms.